modules/xfem/src/userobjects/MeshCut2DRankTwoTensorNucleation.C

```diff
@@ -23,6 +24,15 @@ MeshCut2DRankTwoTensorNucleation::validParams()
   InputParameters params = MeshCut2DNucleationBase::validParams();
   params.addClassDescription(
       "Nucleate a crack in MeshCut2D UO based on a scalar extracted from a RankTwoTensor");
+  params.addRangeCheckedParam<Real>(
+      "edge_extension_factor",
+      1e-5,
+      "edge_extension_factor >= 0",
+      "Crack length scaling factor to extend the nucleated crack beyond the cut element edges.");
```

Comment:
```suggestion
      "Factor by which the length of cracks that have been adjusted to coincide with element edges are increased to avoid missing an intersection with the edge due to numerical tolerance issues."); 
```